### ISSUE TITLE
[ISSUE] Login Page and Footer Section improvement

### ISSUE THREAD

**Issue by DivyanshiNarang** [SrijanShovit/carbonops-v2#27]
**Terms and conditions link can be added**
<img width="256" alt="Screenshot 2023-05-27 003707" src="https://github.com/SrijanShovit/carbonops-v2/assets/74779601/44e66c68-c540-42b2-bcb2-ecce4e70692d">

**In Login page, login with Google is not responding**
<img width="137" alt="Screenshot 2023-05-27 003851" src="https://github.com/SrijanShovit/carbonops-v2/assets/74779601/575277d8-b89f-4dc8-9b95-b91012de3661">

**Social media links can be added in the footer section.**
<img width="249" alt="Screenshot 2023-05-26 235923" src="https://github.com/SrijanShovit/carbonops-v2/assets/74779601/36f5c8aa-640e-4320-af2c-267d44c1f75f">

*Please assign these issues to me under GSSoC'23* @SrijanShovit

**Comment by shreya024** [comment]
Login Page UI is being implemented under #20 .Once it is ready,this issue can be used for implementing OAuth/Google Login.
@DivyanshiNarang If you are interested in UI,you can comment on #20 to get assigned or if interested in implementing the functionality for Login with Google,kindly wait till its PR is merged.

**Comment by shreya024** [comment]
@DivyanshiNarang Any updates?

**Comment by DivyanshiNarang** [comment]
Can I solve Google Authentication issue in a separate PR than these linking and styling issues.

**Comment by shreya024** [comment]
> Can I solve Google Authentication issue in a separate PR than these linking and styling issues.

Google Authentication is already done using OAuth
